scripts/jenkins/groovy/buildConfig.groovy

```diff
@@ -306,6 +306,7 @@ class BuildConfig {
       def change = changeEntry.key
       changeType != "D" && // was not deleted
               change.startsWith('h2o-py/') && change.contains("pyunit_") &&
+              // Allow to run "pyunit_*" files inside of "pyunit_*" directory but do not allow e.g. utilsPY.py to run
```

Comment:
ok, but weird to have line comment between &&s : I'd say better but comments before entire condition or end of line like the previous one